andela/wger-space
# #166320284 Set up code climate

**What does this PR do?**

Adds the repository to codeclimate

**Description of Task to be completed?**

- add the repository to codeclimate
- set the default branch to ch-add-codeclimate-badge-166320284(this will later be set to develop when the PR is merged)
- add the badge to the readme file

**How should this be manually tested?**

- visit the ch-add-codeclimate-badge-16632028 branch on github
- Scroll down to the readme file
- there should be a codeclimate badge displaying the maintainability
- click on the badge to view details on codeclimate

**What are the relevant pivotal tracker stories?**

[#166320284](https://www.pivotaltracker.com/story/show/166320284)

**Screenshots**

<img width="1037" alt="codeclimate" src="https://user-images.githubusercontent.com/40593659/58553015-724f3700-821c-11e9-8231-c32f32f70cc6.png">
